src/Compiler/test/Microsoft.NET.Sdk.Razor.SourceGenerators.Tests/TestFiles/RazorSourceGeneratorCshtmlTests/CssScoping/Pages_Index.html

```diff
@@ -0,0 +1,29 @@
+﻿<h1 test-css-scope>heading</h1>
+<header test-css-scope>header</header>
+<thead test-css-scope>table head</thead>
+<form test-css-scope>
+    <select test-css-scope>
+        <option test-css-scope>choose</option>
+    </select>
+</form>
+< test-css-scope div />
+<div test-css-scope
+>multiline</div>
+< test-css-scopediv>bang</div>
```

Comment:
Is this ok? Shouldn't the input for `<!div>bang</div>` result in `<div test-css-scope>bang</div>` ?

here shows how the result instead of `<div>bang</div>` is `< test-css-scopediv>bang</div>` aka `< {css scope}{starttag}>{content}<{endtag}>`

I belive applying the bang `!` should opt out for taghelpers, currently is messing with the tag completely.

Reply:
Yes, it looks like a bug. Seems unrelated to this PR though.